content/reference/sql/server/replication.md

```diff
@@ -4,32 +4,45 @@ title: Replication
 
 # Replication
 
-Dolt supports two forms of
-[replication](../../../concepts/dolt/rdbms/replication.md).
+Dolt supports two forms of [replication](../../../concepts/dolt/rdbms/replication.md) where a Dolt primary sql-server replicates changes to Dolt sql-server replicas. If you're looking to replicate changes from a MySQL primary server to a Dolt replica, check out the guide to [MySQL Replication to Dolt](../../../guides/binlog-replication.md). 
```

Comment:
Think I would introduce this as, "Dolt can replicate between 2 or more Dolt servers, or can be a read-replica for a MySQL server."

Reply:
I would add a brief "Replicating from a MySQL server" section at the end of this doc, mostly just a link to the guide